Remotion/Data/DomainObjects/Utilities/ObjectIDConverter.cs

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Utilities;
```

Comment:
Not sure if we need a new namespace for just this file. With `WebStringConverter` we just placed it by `WebString`. As such, I would suggest moving it to the root too. ping @MichaelKetting

Reply:
Namespace...

I don't think it has to be in the root namespace. 


I suggest we move the converter to the infrastructure namespace.